src/main/java/data/UserComment.java

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+ 
+/** A user's comment, with corresponding user info. */
+public class UserComment {
+  long id;
+  String username;
```

Comment:
For integration with Users feature: Instead of storing data like username, just store the user's id. This is better because users can change their username or location, so just storing the id prevents data from becoming stale.

Reply:
noted! ty

Reply:
Could you please update with user ID already? using username and then moving to user ID seems unnecessary.

Reply:
Should I leave that for a later PR? This PR was submitted before Ryan even submitted his PR for the Users feature, when we hadn't talked about storing the User's ID yet. We were thinking of finishing integration at the end, after all of our features have been merged in, to make sure that they work; if I just change it now, I'm not sure exactly how it'll work with his code.

Reply:
I think it is a small change now, rather than later. If we have decided on using User ID as a string, we should be using that here instead of the username and location. Plus, using these fields (username and location) here already looks like integration with the other feature.

Reply:
Given that this is blocking review/submission for many things I'd personally be ok with submitting this as is and fixing later.